### PR TITLE
C Program to print  m*n matrix in spiral order.

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given a matrix of m x n elements (m rows, n columns), return all elements of the matrix in spiral order.

Sample case 1:

	Input:

	Enter Rows:2
	Enter Column: 2
	1
	2
	3
	4

	Output:

	Entered Matrix:
	1 2 
	3 4

	Spiral Order of the Matrix: 1 2 4 3

Sample Case 2:

	Input:

	Enter Rows:2
	Enter Column: 2
	0
	0
	0
	0

	Output:

	Null Matrix

Time Complexity: O(ROW*COLUMN)
Space Complexity:O(1).

### Checklist

- [Y ] I've read the contribution guidelines.
- [ Y] I've checked the issue list before deciding what to submit.
- [ Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#792 
ProgramOutput

### CODE REVIEW

**Comment by narayan2111** [comment]
Kindly Review this PR. @Abhijit2505 @plazzy99 @iamrajiv @Kajol-Kumari

**Review by vatsalkesarwani12 — APPROVED** [review]
Code works fine for all test case 👍 
Able to merge

**Review by iamrajiv — APPROVED** [review]

File: C/README.md [hunk 1]
```diff
@@ -1,3 +1,75 @@
+
```
Comment (atarax665) [inline C/README.md:1]: You only need to add your program's hyperlink in the README.md file. Check out other PRs for reference.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Code looks good. Do the suggested changes in readme ASAP so that I can merge this PR.

**Comment by narayan2111** [comment]
> Code looks good. Do the suggested changes in readme ASAP so that I can merge this PR.

How will I do that  sir? 
I mean it works fine and i also updated the readme file and also added link of the code.

**Comment by narayan2111** [comment]
Kindly review this PR, it's pending from 4 days @atarax665

**Review by Abhijit2505 — APPROVED** [review]
Code looks good, just make the ReadMe changes asked before.

**Pull request merged**